input/fsh/EX_RadiotherapyCourse.fsh

```diff
@@ -0,0 +1,35 @@
+Instance: fhir-osiris-example-RadiotherapyCourse-1
+InstanceOf: RadiotherapyCourseSummary
+Description: "Example of instance from OSIRIS_pivot_CourseRT.csv, OSIRIS_pivot_EnergyOrIsotopeRT.csv, OSIRIS_pivot_DoseToVolumeRT.csv"
+
+* identifier.value = "1"
+* subject = Reference(fhir-osiris-example-patient)
+* status = #completed
+* category = RadiotherapyCS#planned
+* performedPeriod.start = "2018-08-15"
+* performedPeriod.end  = "2018-12-25"
+* extension[radiotherapy-course-settings].extension[treatmentIntent].valueCodeableConcept.coding = #373808002
```

Comment:
est ce bien la valeur attendue pour la variable Variable to check: extension[treatmentIntent] (Objectif) ? issue 228